frontend/src/services/historyService.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios"
+const baseUrl = 'http://localhost:8004'
+
+const createHistory = async (historyObject) => {
+    const res = await axios.post(`${baseUrl}/create-history`, historyObject)
+    console.log(res)
+    return res
+}
+
+const getHistory = async (username) => {
+    const res = await axios.get(`${baseUrl}/get-history/${username}`)
+    return res.data
+}
```

Comment:
Can you take a look at #44 and use `HISTORY_SERVICE_URL` instead of baseUrl instead?

Reply:
You could merge main into this branch then update this.

Reply:
I'll submit another PR to change to that method, along with tests, CI and docker